File: matiasmasso/Mat.NET/DT2/DTOSegSocialGrup.vb
Public Class DTOSegSocialGrup
    Inherits DTOBaseGuid

    Property nom As String
    Property id As Integer

    Public Sub New()
        MyBase.New
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub

End Class

File: matiasmasso/Mat.NET/DTO/DTOSegSocialGrup.cs
using System;

namespace DTO
{
    public class DTOSegSocialGrup : DTOBaseGuid
    {
        public string nom { get; set; }
        public int id { get; set; }

        public DTOSegSocialGrup() : base()
        {
        }

        public DTOSegSocialGrup(Guid oGuid) : base(oGuid)
        {
        }
    }
}
